SCClassLibrary/Common/Core/Symbol.sc

```diff
@@ -231,20 +231,20 @@ Symbol {
 
 	archiveAsCompileString { ^true }
 
-	kr { | val, lag, fixedLag = false |
-		^NamedControl.kr(this, val, lag, fixedLag)
+	kr { | val, lag, fixedLag = false, spec |
+		^NamedControl.kr(this, val, lag, fixedLag, spec)
 	}
 
-	ir { | val |
-		^NamedControl.ir(this, val)
+	ir { | val, spec |
+		^NamedControl.ir(this, val, spec:spec)
```

Comment:
passing parameters by keyword is less efficient. In this case it would be better avoided, I think.

Reply:
It can only be avoided in the `ar` case (fixing) - the other cases don't forward a `lags` argument so we've got to specify `spec` by key. It might be semantically identical to pass a nil, but that's really not a common pattern in sclang and in any case, this is not performance-critical code.

Reply:
ok, that makes sense.